src/register-evm-agent/dfx.json

```diff
@@ -6,8 +6,8 @@
             "wasm": ".artifact/simple_canister.wasm",
             "type": "custom"
         },
-        "evmc": {
-            "candid": "./src/simple_canister/evmc.did",
+        "evm": {
+            "candid": "./src/simple_canister/evm.did",
```

Comment:
can we use a full name here, like evm_canister?

Reply:
yes, but yesterday we decided to use only evm